src/components/tab-side-menu/logo.tsx

```diff
@@ -5,28 +5,44 @@ import CompactLogo from "@/assets/logo/compact_logo.svg";
 import MainLogo from "@/assets/logo/main_logo_small.svg";
 import { IsTabMinimizedProps } from "@/types/tab-side-menu";
 
-const logoWrapStyle = cva("", {
+const MainLogoStyle = cva("transition-all absolute duration-300 top-0", {
   variants: {
     open: {
-      true: "hidden md:block lg:block",
-      false: "",
+      true: "translate-x-0 opacity-100",
+      false: "-translate-x-full opacity-0",
     },
   },
   defaultVariants: {
     open: false,
   },
 });
+const CompactLogoStyle = cva(
+  "hidden md:block transition-all left-[50%] absolute duration-500 top-0",
+  {
+    variants: {
+      open: {
+        true: "-translate-x-[50%] opacity-100",
+        false: "-translate-x-500 opacity-0",
+      },
+    },
```

Comment:
```suggestion
const CompactLogoStyle = cva(
  "hidden md:block transition-all left-[50%] absolute top-0",
  {
    variants: {
      open: {
        true: "-translate-x-[50%] duration-500 opacity-100",
        false: "-translate-x-500 duration-0 opacity-0",
      },
    },
```

나타날 때와 사라질 때 애니메이션을 다르게 적용하면 조금 더 자연스러운 애니메이션 효과가 나타납니다. 이건 예시일 뿐이고, 태근님께서 필요하다 생각하는 곳에 추가해보시면 좋을 것 같습니다.